augur/export_v2.py

```diff
@@ -740,6 +741,25 @@ def attr_confidence(
     warn(f"[confidence] {key+'_confidence'!r} is of an unknown format. Skipping.")
     return {}
 
+def valid_url(url: str) -> bool:
+    # <https://stackoverflow.com/questions/7160737/how-to-validate-a-url-in-python-malformed-or-not>
+    try:
+        result = urlparse(url)
+        return all([result.scheme, result.netloc])
+    except AttributeError:
+        return False
+
+def encode_node_attr(value):
+    """Returns a node attr structure with keys 'value' and (optional) 'url'"""
+    if type(value) is not str: # only string values can encode URLs
+        return {"value": value}
+    elif m:=re.match(r'(.+?)\s?<(.+)>$', value): # value <URL>
+        if valid_url(m.groups()[1]):
+            return {"value": m.groups()[0], "url": m.groups()[1]}
+    elif m:=re.match(r'(.+?):\s?(.+)$', value): # value: URL
+        if valid_url(m.groups()[1]):
+            return {"value": m.groups()[0], "url": m.groups()[1]}
```

Comment:
Similar to [my comment](https://github.com/nextstrain/rsv/pull/94#discussion_r2214163028) on using metadata.tsv with the `<value>: <url>` format, I'm not sure how user friendly it is make users encode their URLs in the metadata. 

I think it'd be better to pursue some CLI option/auspice_config configuration for matching values to their urls.